server/views/applications/pages/location-factors/describe-location-factors.njk

```diff
@@ -13,7 +13,9 @@
           classes: "govuk-label--m"
         },
         hint: {
-          text: "For example SW1."
+          html: '<p>For example SW1.</p>
+          <p>We aim to accommodate someone within 50 miles of the postcode area provided.</p>
+          <p></p>'
```

Comment:
Is there a reason for this empty paragraph here?

Reply:
That is meant to have content in it. Oh dear. I'll add it in